src/components/Filters/Filters.tsx

```diff
@@ -35,28 +36,42 @@ const FiltersContainer = styled(FlexContainer)`
 
 const Filters: React.FC<FiltersProps> = ({
   fields,
-  state = getDefaultState(fields),
+  state,
   onApply,
   onChange,
   onClose,
   onCancel,
   isLoading = false,
 }) => {
-  const [filtersValues, setFiltersValues] = useState<Array<Filter>>(state);
+  const [filtersValues, setFiltersValues] = useState<Array<Filter>>(undefined);
```

Comment:
'null' would be more appropriate for default state. 'undefined' is more generic and you don't know if it was setup by you or by the app. On the other hand when you use 'null' you are pretty sure it wasn't generated by application

Reply:
Good point! It is a  good practice, will change it